content/development/crs_toolchain.md

```diff
@@ -1,43 +1,46 @@
 ---
 title: crs-toolchain
-weight: 10
+weight: 12
 disableToc: false
 chapter: false
 ---
 
-> The crs-toolchain is the utility belt of CRS developers. It provides a single point of entry and a consistent interface for a range of different tools. Its core functionality (owed to the great [rassemble-go](https://github.com/itchyny/rassemble-go) (which is itself based on the brain-melting [Regexp::Assemble](https://github.com/ronsavage/Regexp-Assemble) Perl module) is to assemble individual parts of a regular expression into a single expression (with some optimizations).
+> The crs-toolchain is the utility belt of CRS developers. It provides a single point of entry and a consistent interface for a range of different tools. Its core functionality (owed to the great [rassemble-go](https://github.com/itchyny/rassemble-go), which is itself based on the brain-melting [Regexp::Assemble](https://github.com/ronsavage/Regexp-Assemble) Perl module) is to assemble individual parts of a regular expression into a single expression (with some optimizations).
```

Comment:
The comma replaced an opening parenthesis, but the closing one is still there.

Reply:
There was one extra opening bracket before. I think the amendment balances out the brackets? But I hope I haven't accidentally changed the meaning of what you wrote.

Removing the hyperlink syntax to make it easier to read, we have:

> Its core functionality (owed to the great rassemble-go, which is itself based on the brain-melting Regexp::Assemble Perl module) is to assemble individual parts of a regular expression into a single expression (with some optimizations).

I think that's correct. But again, I hope it hasn't altered your original meaning.

Reply:
Oh, yeah, sorry. that opening parenthesis was actually wrong.